.esh/commands/reshape_i18n_keys.py

```
@@ -55,7 +56,14 @@ def dumps_locale(data) -> str:
 	return json.dumps(data, ensure_ascii=False, indent="\t") + "\n"
 
 
+def is_screaming_snake_key(key: str) -> bool:
+	return bool(SCREAMING_SNAKE_RE.match(key))
+
+
 def camel_prefixes(key: str) -> list[str]:
+	# SEND_MESSAGES 等常量键不当驼峰簇成员，避免嵌成 sEND_MESSAGES / mANAGE_.cHANNELS
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)